File: angular/src/app/cards/classes/section.ts
import { Chicklets } from "./chicklets";

export class Section {
    constructor(private sectionId: string,
        private userId: string,
        private operationType: string,
        private chicklets: Chicklets[]) {
        this.sectionId = sectionId;
        this.userId = userId;
        this.operationType = operationType;
        this.chicklets = chicklets;
    }
}

File: ExperienceService/src/main/java/com/stackroute/experienceservice/domain/Section.java
package com.stackroute.experienceservice.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Section {
    private String sectionId;
    private String userId;
    private String operationType;
    private Chicklets chicklets[];
}
